similar_dev_search/dev_stats.py

```diff
@@ -0,0 +1,173 @@
+from collections import defaultdict
+from dataclasses import dataclass
+
+from dulwich.repo import Repo
+
+from extract import get_repo_changes
+import setup
+from utils import get_ngrams, get_ngrams_dict_overlap, get_sorted_dict_by_value, \
+    get_top_k_from_devs_by_metric
+
+
+class DevStats:
+    def __init__(self,
+                 languages=None,
+                 variables=None,
+                 classes=None,
+                 functions=None):
+        if functions is None:
+            functions = defaultdict(float)
+        if classes is None:
+            classes = defaultdict(float)
+        if variables is None:
+            variables = defaultdict(float)
+        if languages is None:
+            languages = defaultdict(float)
+        self.languages = languages
```

Comment:
```suggestion
        return_if_exist = lambda x: x if x else defaultdict(float)
        self.languages = return_if_exist(languages)
```
you can simplify a bit - optional